src/SFA.DAS.RoATPService.Application.Api/Controllers/DownloadRegisterController.cs

```diff
@@ -4,21 +4,19 @@
 namespace SFA.DAS.RoATPService.Application.Api.Controllers
 {
     using System.Collections.Generic;
-    using System.Data.SqlClient;
     using System.Net;
     using System.Threading.Tasks;
-    using Microsoft.AspNetCore.Authorization;
+    using Api.Helpers;
+    using Interfaces;
     using Microsoft.AspNetCore.Mvc;
+    using Microsoft.Data.SqlClient;
     using Microsoft.Extensions.Logging;
-    using OfficeOpenXml;
-    using Api.Helpers;
     using Middleware;
-    using Interfaces;
-    using Swashbuckle.AspNetCore.SwaggerGen;
+    using OfficeOpenXml;
 
-    [Authorize(Roles = "RoATPServiceInternalAPI")]
+    [ApiController]
```

Comment:
is it definitely ok to remove the authorize role here?

Reply:
Roles are applied globally now, so no need to assign it on each controller